tdrs-backend/tdpservice/users/views.py

```diff
@@ -71,7 +71,7 @@ def retrieve(self, request, pk=None):
         return Response(serializer.data)
 
     @action(methods=["GET", "PATCH"], detail=False)
-    def set_profile(self, request, pk=None):
+    def set_profile(self, request):
```

Comment:
these methods only allow the requesting user to update themselves, so i removed the unused `pk` to hopefully clear up any confusion. i'm not sure swagger ever reported it as used, though